six/modules/c++/six.sidd/include/six/sidd/Display.h

```diff
@@ -58,6 +58,18 @@ struct Remap
 
     virtual Remap* clone() const = 0;
 
+    friend bool operator==(const Remap& lhs, const Remap& rhs)
```

Comment:
So I see we are stuck here in terms of `Remap` does need an `equalTo()` virtual method but why are these friend operators needed in the Display class?